packages/frontend/amp/components/Header.tsx

```diff
@@ -200,12 +201,13 @@ export const Header: React.FC<{
                         ${screenReaderOnly};
                     `}
                 >
+                    <AdUserSync />
```

Comment:
inside the A, is that the best place?

Reply:
Agreed  that this would be better else where, why not put it next to `<AdConsent />` in [Article.tsx](https://github.com/guardian/dotcom-rendering/blob/71beb5767f50ab20d271dd355ec33a9909792c2c/packages/frontend/amp/pages/Article.tsx#L79). Seems much more logical to group the ad-related components together.